doc/manuals/user/forbidden_characters.md

```diff
@@ -57,15 +57,17 @@ NGSIv2 introduces syntax restrictions for ID fields (such as entity id/type, att
 or metadata name/type) which are described in the "Field syntax restrictions" section in the
 [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable).
 
-## Custom payload special treatment
+## Custom payload and heaers special treatment
```

Comment:
```suggestion
## Custom payload and headers special treatment
```

Reply:
Fixed in [f75b7c6](https://github.com/telefonicaid/fiware-orion/pull/4072/commits/f75b7c6c374a09407eec72b867f7f377a0f0ead3)